generators/client/templates/angular/src/main/webapp/app/polyfills.ts.ejs

```diff
@@ -1,42 +1,3 @@
-<%#
```

Comment:
@wmarques Did you remove by mistake the copyright

Reply:
oh yes, thanks ! 😅